MAINTAINERS.yml

```diff
@@ -1157,6 +1157,20 @@ Base OS:
     labels:
         - "area: Base OS"
 
+Laird Connectivity platforms:
+    status: maintained
+    maintainers:
+        - greg-leach
```

Comment:
please add just one maintainer and everyone else as collaborators. We are in the process of cleaning up the file to follow this rule.

Reply:
Done